### DISCUSSION TITLE
Orico hub with PD to the host?

### DISCUSSION THREAD

**Discussion by sejtam** [outdoorbits/little-backup-box#239] [Q&A]
So I have the recommended Orico hub to power the drives while copying,  which works fine with the lbb/rpi

But when I want to use it with my MacBook 12, it does not also power the Mac,  so I have to run that off battery (yes,  that Mac only has a single usb-c port,  big mistake, but it is nicely small)

So does anyone know whether a hub exists that would just like the Orico (iepowers several drives from an external power source, and also provides power or its own usb-c input cable to the host?

**Comment by mpwhiten** [comment]
What Orico hub specifically are you using? When I search "Orico powered hub" there are multiple results. It's difficult to help without knowing.

Is the hub USB-C? To power your MacBook, you will need a USB-C PD compatible hub. But if it's USB-C you will have to think about how to connect it to the rpi. Not all USB-A to USB-C adapters work at USB 3.0 speeds, so you would need to carefully select your adapter if you don't want to impact your transfer speeds.
